source/devices/AM62LX/linux/Release_Specific_Release_Notes.rst

```diff
@@ -173,3 +173,37 @@ Yocto
 | Tag Description: SITARA MPU Processor SDK 11.00 Release Tag for AM62L
 |
 
+Issues Tracker
+==============
+
+.. note::
+
+    - Release Specific Issues including details will be published through Software Incident Report (SIR) portal
+
+    - Further Information can be found at `SIR Portal <https://sir.ext.ti.com/>`_
+
+
+Issues Resolved
+---------------
+.. csv-table::
+   :header: "Record ID", "Title"
+   :widths: 15, 70
+
+   "`EXT_EP-12673 <https://sir.ext.ti.com/jira/browse/EXT_EP-12673>`_","AM62L NETDEV WATCHDOG timeout when only one core enabled"
```

Comment:
```suggestion
   "`EXT_EP-12673 <https://sir.ext.ti.com/jira/browse/EXT_EP-12673>`__","AM62L NETDEV WATCHDOG timeout when only one core enabled"
```

I don't see a reason for these to be named links at the moment. Nothing seems to be referencing these issues directly yet, but maybe that will be useful going forward. Open to opinions here.